controllers/articles.controller.js

```diff
@@ -0,0 +1,17 @@
+const { requireArticles } = require("../models /articles.model");
+const { requireArticleById } = require("../models /articles.model");
+
+exports.getArticles = (req, res) => {
+  requireArticles().then((data) => {
+    res.status(200).send({ data: data });
```

Comment:
This isn't the ticket we're reviewing. Even then, the object should have a key of articles